ui/src/app/components/sidenav-review-changes/sidenav-review-changes.component.ts

```diff
@@ -61,7 +61,11 @@ export class SidenavReviewChangesComponent implements OnInit {
       .subscribe({
         next: (res: string) => {
           if (res == '') {
-            this.snackbar.openSnackBar('Table updated successfully', 'Close', 5)
+            this.snackbar.openSnackBar(
+              `Schema changes to table ${this.tableUpdateData.tableName} saved successfully`,
```

Comment:
In case of interleaved table, its still showing only one table name. It might confuse the users

Reply:
Updated